routing/openssl.conf

```diff
@@ -0,0 +1,21 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+# disable prompting of certificate fields and just takes values from the config file directly
+prompt = no
+
+[req_distinguished_name]
+C = US
+ST = VA
+L = Somewhere
+O = RedHat
+OU = OpenShift QE
+CN = apps
+
+[v3_req]
+basicConstraints = CA:FALSE
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1 = *.apps.your-cluster.qe.devcluster.openshift.com
```

Comment:
@qinpingli @jhou1 Are we OK to expose the URL ?

Reply:
that's just an example, let me update to `example.com`.

Reply:
Removed this line and just kept `*.example.com`.